package.json

```diff
@@ -18,6 +18,7 @@
     "keywords": [
         "rsa",
         "blindrsa",
+        "signature",
```

Comment:
I confirm the tag seems relevant based on other published packages https://www.npmjs.com/search?q=keywords:signature